cocotb/share/makefiles/Makefile.sim

```diff
@@ -89,15 +101,16 @@ SIM ?= icarus
 SIM_LOWERCASE := $(shell echo $(SIM) | tr A-Z a-z)
 
 # Directory containing the cocotb Makfiles (realpath for Windows compatibility)
-COCOTB_MAKEFILES_DIR := $(realpath $(shell cocotb-config --makefiles))
+COCOTB_MAKEFILES_DIR := $(realpath $(shell $(COCOTB_CONFIG_CMD) --makefiles))
```

Comment:
Is there any advantage to changing this line?

Reply:
... compared to creating a COCOTB_CONFIG_CMD variable, but not using it? No if you're looking purely at functionality, yes if you're looking at cleanliness.